tests/Aql/ExecutorTestHelper.h

```diff
@@ -37,6 +44,189 @@ namespace arangodb {
 namespace tests {
 namespace aql {
 
+template <typename E, std::size_t inputColumns = 1, std::size_t outputColumns = 1>
+struct ExecutorTestHelper {
```

Comment:
\o/

This is very useful to write specific and also short tests.
I think this will develop some more over time